style.css

```diff
@@ -720,3 +744,457 @@ textarea {
+    justify-items: center;
+  }
+
+  [class*="nav-"] {
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    color: #344563;
+  }
+
+  .hamburger-container {
+    display: none;
+  }
+
+  .intro-section {
+    width: 100%;
+    height: 916px;
+    left: 0;
+    top: 71px;
+    padding: 255px 18% 255px 18%;
+  }
+
+  .top-heading {
+    font-weight: 700;
+    font-size: 48px;
+    line-height: 60px;
+  }
+
+  .heading-break {
+    display: none;
+  }
+
+  .middle-section {
+    width: 100%;
+    height: 2694px;
+    left: 0;
+    top: 917px;
+    padding: 5.3% 12.5% 5.3% 12.5%;
+    row-gap: 142px;
+  }
+
+  .first-project,
+  .second-project,
+  .third-project,
+  .fourth-project {
+    position: relative;
+    background: #fff;
+    border: 1px solid #dfe1e6;
+    border-radius: 16px;
+    width: 80vw;
+    padding: 24px;
+    padding-right: 48px;
+    display: grid;
+    grid-template-columns: 50% 50%;
+    column-gap: 22px;
+  }
+
+  .first-project-work-image,
+  .second-project-work-image,
+  .third-project-work-image,
+  .fourth-project-work-image {
+    order: 0;
+    width: 100%;
+    height: 100%;
+  }
+
+  .dcw2,
+  .dcw4 {
+    order: 2;
+  }
+
+  .details-container-desktop {
+    order: 1;
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    height: 100%;
+    padding: 24px;
+    gap: 24px;
+  }
+
+  .details-container {
+    display: none;
+  }
+
+  .project-heading {
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+    color: #091e42;
+  }
+
+  .project-type {
+    order: -2;
+    font-weight: 500;
+    font-size: 18px;
+    line-height: 24px;
+    color: #344563;
+  }
+
+  .which-end {
+    order: 0;
+    font-weight: 400;
+    font-size: 18px;
+    line-height: 24px;
+    color: #6b778c;
+  }
+
+  .project-year {
+    order: 2;
+    font-weight: 400;
+    font-size: 18px;
+    line-height: 24px;
+    color: #6b778c;
+  }
+
+  .project-decription {
+    width: 295px;
+    height: 76px;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+    color: #344563;
+  }
+
+  .ruby-on-rails {
+    float: left;
+    margin-right: 12px;
+    padding: 4px 12px;
+    width: 106px;
+    height: 24px;
+    background: #ebebff;
+    border-radius: 8px;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 16px;
+    color: #6070ff;
+  }
+
+  .project-button {
+    position: relative;
+    top: 10px;
+  }
+
+  .project-button:hover {
+    width: 124px;
+    height: 48px;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    gap: 10px;
+    box-shadow: 0 8px 16px rgba(64, 83 252, 0.24);
+    border-radius: 8px;
+  }
+
+  .project-text:hover {
+    position: relative;
+    width: 124px;
+    height: 48px;
+    padding: 12px;
+    background: #6070ff;
+    top: calc(50% - 48px / 2);
+    border: 1px solid #6070ff;
+    border-radius: 8px;
+    font-weight: 500;
+    font-size: 17px;
+    line-height: 24px;
+    color: #fff;
+  }
+
```

Comment:
Well done adding the button interactions 👍 
- [ ] I can see that you have added the `hover` and `active` interaction to both `button` and `text` because of which when I'm pressing a `button` there is a `box-shadow` displaying along `button`. I think it would be better if you follow the figma template and add the correct interactions. Right now, because of adding interactions to `text` and `button`, there is `box-shadow`. See the video below for more details 🔽
- [ ] Also, there is slight displacement of button while hovering over it. I think it would be better if you can try fixing this, because there is no displacement of button present in figma design. One way of doing this is simply removing `height` from `project-button` class. However, you are open to use your own styling. I know you can do that 💪

```
.project-button {
    /* height: 72px; */
}

```

https://user-images.githubusercontent.com/106119105/214986747-efc83f60-0420-4e23-b13b-9c4d28b12fd0.mp4

Reply:
Could you please further elaborate the suggestion regarding box-shadow? I can not understand your point.

Reply:
Hello @sameedmohsin28 👋
Yes of course, I am happy to help
You are using two classes to style the `presed/active` button . The first one id `.project-button:active` and the other is `.project-text:active`. Right? And if you see the `.project-button:active` has `box-shadow: 0 8px 16px;` property. So when I'm clicking on the button there is a box-shadow (as shown in video 👆) which is not present according to the figma template. It can be fixed by simply removing `box-shadow` property. But please be careful that it does not effect your styling 😃 Is it a bit clear? Please let me know if it's not 💪
```
.project-button:active {
    width: 124px;
    height: 48px;
    display: flex;
    flex-direction: row;
    align-items: center;
    gap: 10px;
    box-shadow: 0 8px 16px;
    border: rgba(64, 83, 252, 0.24);
    border-radius: 8px;
  }

  .project-text:active {
    position: relative;
    width: 124px;
    height: 48px;
    padding: 12px;
    background: #2230d2;
    top: calc(50% - 48px / 2);
    border: 1px solid #2230d2;
    border-radius: 8px;
    font-weight: 500;
    font-size: 17px;
    line-height: 24px;
    color: #fff;
  }
```

Reply:
Yes, it has helped. Thanks alot!